### ISSUE TITLE
simple-sendfile-0.2.18 test suite failure

### ISSUE THREAD

**Issue by peti** [kazu-yamamoto/simple-sendfile#17]
The latest version of `simple-sendfile` does not succeed its test suite when run in our automated CI system: http://hydra.cryp.to/build/497821/nixlog/1/raw. Do you have any suggestion what might be the cause for these errors and what we could do to fix them? Our solution right now is to just not run the test suite, which seems unsatisfactory.

**Comment by kazu-yamamoto** [comment]
This is because `test/inputFile` exists in the github repo but is not included in a package. I would not like to include it since it is huge and unnecessary for users. You could just copy a big local file to the `test` directory to pass the test. @peti What do you think?

**Comment by peti** [comment]
Yes, I suppose we can do that. I'm not sure it's worthwhile, though. If users aren't supposed to run the test suite from the release tarball, than I reckon it's fine if we don't run the test suite in Nix builds either.

**Comment by nomeata** [comment]
I was bitten by this. Have you considered generating the `test/inputFile` in the testsuite?

**Comment by kazu-yamamoto** [comment]
It's nice if the file is generated. A patch is welcome.
